class/defaults.yml

```
@@ -14,6 +14,7 @@ parameters:
         registry: docker.io
         repository: projectsyn/steward
         version: v0.3.1
+    componentPath: 'dependencies/lieutenant' # Only changed for testing
```

If you want to go ahead with this parameter, please mark it as "internal", by prefixing it with an underscore.

```suggestion
    _componentPath: 'dependencies/lieutenant' # Only changed for testing
```